docker-compose.yml

```diff
@@ -400,6 +420,8 @@ volumes:
     external: true
   sentry-symbolicator:
     external: true
+  sentry-vroom:
+    external: true
```

Comment:
Don't forget to add a line about this here: https://github.com/getsentry/self-hosted/blob/master/install/create-docker-volumes.sh